packages/zowe-explorer/src/job/ZoweJobNode.ts

```diff
@@ -274,6 +274,21 @@ export class Job extends ZoweTreeNode implements IZoweJobTreeNode {
                     owner,
                     prefix
                 );
+                /*
```

Comment:
package fails due to whitespace errors.  Update comment starting on line 277 to have `/**` with `*` at beginning of each line and ending line of comment being `*/`

Reply:
<img width="753" alt="Screen Shot 2022-05-09 at 5 33 47 PM" src="https://user-images.githubusercontent.com/49491949/167502471-31ff342a-2f9c-4d67-8235-18f22c02eea1.png">